solutions/sum_numbers.py

```diff
@@ -0,0 +1,61 @@
+""" sum numbers 
+sum numbers is a recursive function return the sum of count down numbers 
+
+created: 25/12/2024
+Team number:28
+Team name:MIT Alpha
+The author: Maab Mohamedkhair"""
+
+
+def sum_numbers(n):
+    
+    """ sum numbers 
+sum numbers is a recursive function return the sum of count down numbers 
+from n -> 1
+
+Base case: n == 1 
+    return 1
+
+recursion case: for n > 1
+    smaller_result of n + the value of sum_numbers(n - 1)
+
+parameters: n = any positive integer less than 1000 because python limitation in recursion 
+
+return : the sum of all numbers less than n inclusive and greater than or equal one
+
+Examples:
+
+>>> sum_numbers(4)
+10
+
+>>> sum_numbers(5)
+15
+
+>>> sum_numbers(0)
+Traceback (most recent call last):
```

Comment:
traceback is an error because of the logic when you pass 0 it will raise exception duo to lines 47 and 48

Reply:
this way of writing the test was the only way for the doc-test to work so I didn't get the logic behind it